bleak/backends/bluezdbus/client.py

```diff
@@ -167,6 +169,7 @@ def on_value_changed(char_path: str, value: bytes) -> None:
                 # For additional details see https://github.com/bluez/bluez/issues/89
                 #
                 if not manager.is_connected(self._device_path):
+                    logger.debug(f"Connecting to BlueZ path {self._device_path}")
```

Comment:
Sure, no problems I'll make this change.

FWIW I did it this way because it seemed to be the most common logging style in this file already (for example lines 131, 224, 269, etc).

Reply:
Yeah, one of these days we need to go through and fix those as well.